cli/events-plugin.md

```diff
@@ -0,0 +1,328 @@
+# Events Plugin for Adobe I/O CLI
+
+This guide includes step by step instructions for setting up and using the Events Plugin for the Adobe I/O CLI.
+
+In addition to the information provided in this guide, you can also visit the [Events Plugin GitHub repository](https://github.com/adobe/aio-cli-plugin-events) or the [Adobe I/O CLI GitHub repository](https://github.com/adobe/aio-cli) to learn more.
+
+* [Setup and credentials](#setup-and-credentials)
+* [Installing and using the plugin](#installing-and-using-the-plugin)
+* [Commands](#commands)
+
+## Setup and credentials
+
+To begin setting up the Events Plugin, visit [Adobe Developer Console](https://www.adobe.com/go/devs_console_ui) and create a new project in your organization. For detailed instructions, follow the steps outlined in this tutorial for [creating a new project](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/projects-empty.md).
+
+> **Note:** Creating a project will also create a Production workspace that will be used later in this workflow when setting up credentials.
```

Comment:
It depends. They can have custom workspaces. Its not always Production workspace. But a workspace will be created. i dont think we need to add this note.